op-chain-ops/genesis/config.go

```diff
@@ -31,6 +31,16 @@ var (
 	ErrInvalidImmutablesConfig = errors.New("invalid immutables config")
 )
 
+// todo(maurelian): Is it possible to reuse the type def in the superchainconfig.go bindings?
+// SequencerKeys is an auto generated low-level Go binding around an user-defined struct.
+type SequencerKeys struct {
```

Comment:
We can definitely reuse the typedef from the bindings, I am not super opinionated on whether we use it here or not

Reply:
I think we should use our own type so that we can add a function to the type to compute the hash

Reply:
Addressed this in the parent PR.